@xen-orchestra/backups/.USAGE.md

```diff
@@ -0,0 +1,12 @@
+## Changing the default backup configuration
+
+It is possible to change the default backup configuration by editing the XO configuration file (`config.toml`).
+
+There are multiple sections concerning backups:
+
+- `[backups]` contains values related to the general backup configuration
+- `[backups.defaultSettings]`, `[backups.vm.defaultSettings]` and `[backups.metadata.defaultSettings]` contain values related to backup jobs default settings:
+  - `[backups.vm.defaultSettings]` is related to VM backup jobs
+  - `[backups.metadata.defaultSettings]` is related to metadata backup jobs
+  - `[backups.defaultSettings]` is related to all backup jobs
+  - in case of duplicate value, `[backups.vm.defaultSettings]` and `[backups.metadata.defaultSettings]` will prevail over `[backups.defaultSettings]`, and a defined job setting will prevail over any `defaultSettings`
```

Comment:
I'm afraid that documentation like this will become obsolete at one point when the code is updated and the doc is not.

But these entries are probably high level enough that it will not happen often.

Reply:
this is always an issue with documentation, but this part is not really discoverable, and created a lot of undue work and support already

Reply:
Do you have any idea to mitigate this issue?
We could maybe add a comment in the code to remember to update the doc if this part changes, but it doesn't look like a good solution